Makefile

```diff
@@ -282,35 +296,46 @@ test-postgis:
 	psql postgresql://postgres:postgres@localhost:5432 -t -c "select 'test postgres';"
 
 .PHONY: test-core
-test-core: .venv/requirements-timestamp
-	$(VENV_BIN)/py.test -vv $(PYTEST_OPTS) --cov-config .coveragerc.core --cov $(PACKAGE)/core --cov-report=term-missing --cov-report=xml:coverage.core.xml tests/core
+test-core: ${VENV_ROOT}/requirements-timestamp
+	$(VENV_BIN)/py.test --pdb -vv $(PYTEST_OPTS) --cov-config .coveragerc.core --cov $(PACKAGE)/core --cov-report=term-missing --cov-report=xml:coverage.core.xml tests/core
 
 .PHONY: test-contrib-print_proxy-mapfish_print
-test-contrib-print_proxy-mapfish_print: .venv/requirements-timestamp
+test-contrib-print_proxy-mapfish_print: ${VENV_ROOT}/requirements-timestamp
 	$(VENV_BIN)/py.test -vv $(PYTEST_OPTS) --cov-config .coveragerc.contrib-print_proxy-mapfish_print --cov $(PACKAGE) --cov-report xml:coverage.contrib-print_proxy-mapfish_print.xml tests/contrib.print_proxy.mapfish_print
 	# $(VENV_BIN)/py.test -vv $(PYTEST_OPTS) --cov-config .coveragerc --cov $(PACKAGE) --cov-report term-missing:skip-covered tests/contrib.print_proxy.xml_2_pdf
 
 .PHONY: test-contrib-data_sources-standard
-test-contrib-data_sources-standard: .venv/requirements-timestamp
+test-contrib-data_sources-standard: ${VENV_ROOT}/requirements-timestamp
 	$(VENV_BIN)/py.test -vv $(PYTEST_OPTS) --cov-config .coveragerc.contrib-data_sources-standard --cov $(PACKAGE)/contrib/data_sources/standard --cov-report=term-missing:skip-covered --cov-report=xml:coverage.contrib-data_sources-standard.xml tests/contrib.data_sources.standard
 
 .PHONY: test-contrib-stats
 test-contrib-stats: .venv/requirements-timestamp
 	$(VENV_BIN)/py.test -vv $(PYTEST_OPTS) --cov-config .coveragerc.contrib-stats --cov $(PACKAGE)/contrib/stats --cov-report=xml:coverage.contrib-stats.xml tests/contrib.stats
 
 .PHONY: tests
-tests: .venv/requirements-timestamp test-core test-contrib-data_sources-standard test-contrib-print_proxy-mapfish_print test-contrib-data_sources-standard test-contrib-stats
+tests: ${VENV_ROOT}/requirements-timestamp test-core test-contrib-data_sources-standard test-contrib-print_proxy-mapfish_print test-contrib-data_sources-standard test-contrib-stats
+
+.PHONY: docker-tests
+docker-tests:
```

Comment:
Minor: About this integrated targets it's ok to have them but this means you need to have Make installed on the machine you issue them which is a bit cat bites own tail. These people have to run the docker-compose commands by hand then as they do now.